app/controllers/banners_controller.rb

```diff
@@ -11,34 +11,44 @@ def new
     authorize :application, :admin_or_supervisor?
 
     @banner = Banner.new
+    @org_has_alternate_active_banner = current_organization.banners.where(active: true).where.not(id: @banner.id).exists?
```

Comment:
You could also consider making this a method:

```
@org_has_alternate_active_banner = current_organization.has_active_banner?(@banner.id)
```

```
# app/models/casa_org

class CasaOrg
  ...

  def has_active_banner?(current_banner_id)
    banners.where(active: true).where.not(id: current_banner_id).exists?
  end
end
```

Reply:
Also if you make it a method, you could probably get away without passing the instance variable around, and just call the method on line 25 of `app/views/banners/_form.html.erb`